pyathena/sqlalchemy_athena.py

```diff
@@ -272,17 +352,21 @@ def post_create_table(self, table):
     "double": FLOAT,
     "tinyint": INTEGER,
     "smallint": INTEGER,
+    "int": INTEGER,
     "integer": INTEGER,
     "bigint": BIGINT,
     "decimal": DECIMAL,
     "char": STRINGTYPE,
     "varchar": STRINGTYPE,
     "array": STRINGTYPE,
     "row": STRINGTYPE,  # StructType
+    "struct": STRINGTYPE,  # StructType
     "varbinary": BINARY,
+    "binary": BINARY,
     "map": STRINGTYPE,
     "date": DATE,
     "timestamp": TIMESTAMP,
+    "string": STRINGTYPE,
```

Comment:
Type names that differs when using the API compared to the information schema.
Should we remove the old ones ?
```suggestion
    "int": INTEGER,
    "bigint": BIGINT,
    "decimal": DECIMAL,
    "char": STRINGTYPE,
    "varchar": STRINGTYPE,
    "array": STRINGTYPE,
    "struct": STRINGTYPE,  # StructType
    "binary": BINARY,
    "map": STRINGTYPE,
    "date": DATE,
    "timestamp": TIMESTAMP,
    "string": STRINGTYPE,
```